src/main/java/lotto/step4/domain/LottoStatistics.java

```diff
@@ -0,0 +1,30 @@
+package lotto.step4.domain;
+
+import java.util.List;
+
+
+public class LottoStatistics {
+    private final List<Rank> statics;
+
+    public LottoStatistics(List<Rank> statics) {
+        this.statics = statics;
+    }
+
+    public long count(Rank rank) {
+        return statics.stream().filter(it -> it == rank).count();
```

Comment:
```suggestion
        return statics.stream()
        .filter(it -> it == rank)
        .count();
```

행위 단위로 내려쓰는것이 가독성과 디버깅에 장점이 만은것 같아요 :)